man/decostand.Rd

```diff
@@ -106,25 +106,37 @@ decobackstand(x, zap = TRUE)
      (e.g. the smallest positive value in the data), either by
      adding it manually to the input data, or by using the argument
      \code{pseudocount} as in
-     \code{decostand(x, method = "clr", pseudocount = 1)}. Adding
+     \code{decostand(x, method = "clr", na.rm=TRUE, pseudocount = 1)}. Adding
      pseudocount will inevitably introduce some bias; see
-     the \code{rclr} method for one available solution.
-
-     \item \code{rclr}: robust clr ("rclr") is similar to regular clr
-     (see above) but allows data that contains zeroes. This method
-     does not use pseudocounts, unlike the standard clr.
-     The robust clr (rclr) divides the values by geometric mean
-     of the observed features; zero values are kept as zeroes, and not
-     taken into account. In high dimensional data,
-     the geometric mean of rclr approximates the true
-     geometric mean; see e.g. Martino et al. (2019)
-     The \code{rclr} transformation is defined formally as follows:
+     the \code{rclr} method for an alternative.
+
+    \item \code{rclr}: robust clr ("rclr") is similar to regular clr
+     (see above) but it allows data with zeroes. This method can avoid
+     the use of pseudocounts, unlike the standard clr. The robust clr
+     (rclr) the logarithmizes the data and divides it by the geometric
+     mean of the observed features within each sample. In high
+     dimensional data the geometric mean of rclr approximates the true
+     geometric mean; see e.g. Martino et al. (2019). The \code{rclr}
+     transformation is defined formally as follows:
      \deqn{rclr = log\frac{x}{g(x > 0)}}{%
-     rclr = log(x/g(x > 0))}
-     where \eqn{x} is a single value, and \eqn{g(x > 0)} is the geometric 
-     mean of sample-wide values \eqn{x} that are positive (> 0).
+           rclr = log(x/g(x > 0))} where \eqn{x} is
+     a single value, and \eqn{g(x > 0)} is the geometric mean of
+     sample-wide values \eqn{x} that are positive (> 0). The OptSpace
+     algorithm performs matrix completion for the missing values
+     that result from log transformation of the zero entries in the
+     original input data. The vegan implementation of the matrix
```

Comment:
I don't think we need to repeat this
>The vegan implementation of the matrix
     completion was adapted from the original implementation
     ROptSpace::OptSpace (version 0.2.3) following Keshavan et
     al. (2010).

here. Just direct people to the OptSpace Rd file with `See \link[vegan]{OptSpace} for more details.` Or something like that.